docs/getting-started/config/authentication.md

```diff
@@ -71,20 +76,58 @@ components: {
+To add the auth provider to the backend we will first need to install the package by running this command:
 
-:::
+```bash title="from your Backstage root directory"
+yarn --cwd packages/backend add @backstage/plugin-auth-backend-module-github-provider
+```
+
+Then we will need to add this line:
 
-Restart Backstage from the terminal, by stopping it with `Control-C`, and starting it with `yarn dev` . You should be welcomed by a login prompt!
+```ts title="in packages/backend/src/index.ts"
+backend.add(import('@backstage/plugin-auth-backend'));
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-auth-backend-module-github-provider'));
+/* highlight-add-end */
+```
+
+Restart Backstage from the terminal, by stopping it with `Control-C`, and starting it with `yarn dev`. You should be welcomed by a login prompt!
 
 :::note Note
 
 Sometimes the frontend starts before the backend resulting in errors on the sign in page. Wait for the backend to start and then reload Backstage to proceed.
 
 :::
 
+If you try to login at this point you will get a "Failed to sign-in, unable to resolve user identity" message, read on as we'll fix that next.
+
+### Adding a User
+
+As part of the authentication process it will try to match details from the Auth Provider User Profile to User in your Catalog. This is what the `resolver` in the config we added to the `app-config.yaml` in the "[Setting up authentication](#setting-up-authentication)" section is doing.
+
+The recommended approach for adding Users, and Groups, into your Catalog is to use one of the existing Org Entity Providers - [like this one for GitHub](https://backstage.io/docs/integrations/github/org) - or if those don't work you may need to [create one](https://backstage.io/docs/features/software-catalog/external-integrations#custom-entity-providers) that fits your Organization's needs.
+
+For the sake of this guide we'll simply step you though adding a User to the `org.yaml` file that is included when you create a new Backstage instance. Let's do that:
+
+1. Fist open the `/examples/org.yaml` file in your text editor of choice
+2. At the bottom we'll add the following YAML:
+
+   ```yaml
+   ---
+   apiVersion: backstage.io/v1alpha1
+   kind: User
+   metadata:
+     name: YOUR GITHUB USERNAME
+   spec:
+     memberOf: [guests]
+   ```
+
+3. Now make sure to replace the text "YOUR GITHUB USERNAME" with your actual GitHub User name.
+
+Let's restart Backstage from the terminal ponce more, by stopping it with `Control-C`, and starting it with `yarn dev`. You should now be able to log into Backstage and see items in you Catalog.
```

Comment:
I think this can also now be done by just pressing enter in the terminal that Backstage is running in?

Reply:
As I'm not sure I'd like to just leave it.